frontend/utils/jwtDecode.ts

```diff
@@ -1,4 +1,4 @@
-import jwt_decode from 'jwt-decode'
+import * as jwt_decode from 'jwt-decode'
```

Comment:
@issiyda 
8行目のjwt_decodeの箇所でエラーが出ております。
```
This expression is not callable.
  Type 'typeof import("/Users/home/practice/engineer-tenshoku-master/frontend/node_modules/jwt-decode/build/cjs/index")' has no call signatures.
```

Reply:
```suggestion
import jwt_decode from 'jwt-decode'
```

元に戻したら行けましたよ！

- ユーザが登録されているか？→AWSのcognito確認
- 新規登録したユーザで再度やってみる

など試してもらえると。